odoo/tools/mail.py

```diff
@@ -239,6 +244,9 @@ def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=Fals
         cleaned = cleaned.replace(u'%7C', u'|')
         cleaned = cleaned.replace(u'&lt;%', u'<%')
         cleaned = cleaned.replace(u'%&gt;', u'%>')
+        # OUTLOOK compatibility: restore mso conditionals
+        cleaned = cleaned.replace(u'<%mso_if%>', u'<!--[if mso]>')
```

Comment:
Also don't forget to add tests to ensure this is effectively working. There are already tests for cleaner so you would probably simply have to add some light unit tests.